build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
@dmydlarz maybe let's just put this into README? What's the point of having this file here?